docs/api/ref/Events.md

```diff
@@ -38,9 +67,24 @@ Used when attaching listeners to events.
 
 ## `FDC3EventTypes`
 
+<Tabs groupId="lang">
+<TabItem value="ts" label="TypeScript/JavaScript">
+
 ```ts
 type FDC3EventTypes = "userChannelChanged";
 ```
+</TabItem>
+<TabItem value="dotnet" label=".NET">
+
+```csharp
+enum FDC3EventTypes 
```

Comment:
Yes, this differs from the Typescript one, but it is a different language and expectations.  This can be discussed.  I'm torn on even having the enum and just going with a string type that has public readonly strings exposed for comparisons.

Reply:
As you are designing a binding for a specified language I personally think it makes sense to use naming, casing and API conventions for that language. The common concept remains easily recognizable across languages.

Reply:
Went through and added all the other suggestions as I had only marked one pending discussion.